SQL/Cleanup_patches/2020-05-12-ChangeCandidateEmails.sql

```diff
@@ -0,0 +1,4 @@
+--Changes survey accounts emails to person[1...]@example.com
```

Comment:
This patch is dangerous!!

I'm guessing this is what you used on the RB dataset to fix the emails. Now imagine we merge it and a real project runs this patch on their database. It will effectively erase all the real data and replace it with fake emails... Although we do want that for RB, we really don't for IBIS, CCNA, ...

So basically please remove this file

Reply:
For future reference, do changes to the RB database need a (cleanup) patch file for existing instances?

Reply:
I would say it's the other way around.

In this case, we identified a flaw with RB and needed to fix it. this is not an issue for projects with active data. so the PR should only address the RB data.

In other cases, for example if we find a bug in the software, then a patch needs to be created for ALL projects (including live datasets) and we treat RB as one of them so we apply the patches on RB as well to keep our mock dataset up to date.

Quick example, let's say that the previous release of loris had an issue where it would automatically append a `+` sign to all user's first names. to fix the issue, we would fix the bug in the code and subsequently issue an SQL patch to "undo" the effects of this bug, so the patch would remove the appended `+` from all first names. In that case we would run that patch on RB to make sure our mock dataset also gets cleaned out of erroneous data